lightstepoc/internal/conversions/package.go

```diff
@@ -7,31 +7,19 @@ import (
 	"go.opencensus.io/trace"
 )
 
-func ConvertTraceID(original trace.TraceID) (uint64, bool) {
-	if traceID, n := binary.Uvarint(original[8:]); traceID != 0 && n > 0 {
-		return traceID, true
-	}
-	return 0, false
+func ConvertTraceID(original trace.TraceID) uint64 {
+	return binary.BigEndian.Uint64(original[:])
```

Comment:
This should use `original[:8]` I think.  (Are we sure the other OC->LS exporters are behaving exactly the same?)

Reply:
I believe this is at least the behaviour in Java, per [this line](https://github.com/lightstep/lightstep-census-java/blob/540c182e1651bfd336085292a717b82add75d89d/src/main/java/com/lightstep/opencensus/exporter/LightStepExporterHandler.java#L168).